engine/Dockerfile

```diff
@@ -29,6 +29,7 @@ RUN chown -R 1000:2000 /tmp/silk_profiles
 
 FROM base AS dev
 RUN apt-get install -y sqlite3 default-mysql-client postgresql-client
+CMD [ "uwsgi", "--ini", "uwsgi.ini" ]
```

Comment:
I _think_ if we move this up into the `base` image layer it'd avoid the need to duplicate w/ line 44